source/deployment/deployment.md

```diff
@@ -65,7 +65,7 @@ The [Mattermost Push Notification Service (MPNS)](http://docs.mattermost.com/dep
 
 If you're deploying mobile applications to an Enterprise App Store, your MPNS should be behind your firewall on your private network. If you're using mobile apps in iTunes and Google Play, you can relay notifications to mobile apps using the [Hosted Push Notification Service (HPNS)](http://docs.mattermost.com/deployment/push.html#hosted-push-notifications-service-hpns) included with Mattermost Enterprise Edition. 
 
-HPNS does not connect to your mobile apps directly, it send messages over an encrypted channel to Apple or Google to relay to the apps users download from iTunes or Google Play. 
+HPNS does not connect to your mobile apps directly, it send messages over an encrypted channel to Apple or Google which are relayed to the apps users download from iTunes or Google Play. 
```

Comment:
"it send messages" > "it sends messages"

Still not sure about "to the apps users download" - is it trying to say "to the app users downloaded from iTunes or Google Play"?